Bugzilla/WebService/Bugzilla.pm

```diff
@@ -80,6 +83,36 @@ sub time {
     };
 }
 
+sub jobqueue_status {
+    my ( $self, $params ) = @_;
+    
+    Bugzilla->login(LOGIN_REQUIRED);
+
+    my $dbh = Bugzilla->dbh;
+    my $query = qq{
```

Comment:
use `q{` in place of `qq{` here since you don't interpolate any variables into the string.